storage/sealer/sched.go

```diff
@@ -168,7 +177,8 @@ func newScheduler(assigner string) (*Scheduler, error) {
 			prepared: map[uuid.UUID]trackedWork{},
 		},
 
-		info: make(chan func(interface{})),
+		info:      make(chan func(interface{})),
+		rmRequest: make(chan *rmRequest, 1),
```

Comment:
This channel doesn't really need to be buffered, it doesn't change how much waiting we're doing, and it may even lead to a case where we put the request here, then cancel the RemoveRequest context, and the response has nowhere to go.